python/ray/includes/unique_ids.pxd

```diff
@@ -26,14 +27,97 @@ cdef extern from "ray/id.h" namespace "ray" nogil:
         c_string binary() const
         c_string hex() const
 
-ctypedef CUniqueID CActorCheckpointID
-ctypedef CUniqueID CActorClassID
-ctypedef CUniqueID CActorHandleID
-ctypedef CUniqueID CActorID
-ctypedef CUniqueID CClientID
-ctypedef CUniqueID CConfigID
-ctypedef CUniqueID CDriverID
-ctypedef CUniqueID CFunctionID
-ctypedef CUniqueID CObjectID
-ctypedef CUniqueID CTaskID
-ctypedef CUniqueID CWorkerID
+    cdef cppclass CActorCheckpointID "ray::ActorCheckpointID"(CUniqueID):
+        CActorCheckpointID()
+        CActorCheckpointID(const CUniqueID &from_id)
```

Comment:
it seems that these constructors aren't used in Cython. We can remove them to minimize redundant code.

Reply:
I removed both constructors for each cppclass.